### ISSUE TITLE
Have separate sync log tabs for uploads, downloads, deletions

### ISSUE THREAD

**Issue by sagamusix** [owncloud/client#5089]
It would be nice to (optionally?) divide the synchronization log into three tabs: Uploaded files, downloaded files and deleted files. That way, it would be easier to review the log in case any unwanted actions happened (e.g. files being deleted locally).

Alternatively, the headers of the log's list view could be made clickable so that you can sort by action rather than by date.

**Comment by dragotin** [comment]
I agree to your proposal to make the columns sortable. Also we will add a filter-as-you-type filter entry-field.
